xml/templates/civicrm_data.tpl

```diff
@@ -284,6 +285,8 @@ SELECT @option_group_id_ps    := max(id) from civicrm_option_group where name =
 SELECT @option_group_id_crs    := max(id) from civicrm_option_group where name = 'contribution_recur_status';
 SELECT @option_group_id_env    := max(id) from civicrm_option_group where name = 'environment';
 SELECT @option_group_id_default_assignee := max(id) from civicrm_option_group where name = 'activity_default_assignee';
+SELECT @option_group_id_default_assignee := max(id) from civicrm_option_group where name = 'activity_default_assignee';
```

Comment:
@eileenmcnaughton this looks like it is duplicated in this line

Reply:
thanks @seamuslee001 I removed it - but I think there is something else wrong in here. So far I've tested the upgrade locally but `civibuild create` - so it's probably something obvious